stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -55,6 +55,10 @@ typename return_type<T_x, T_alpha, T_beta>::type bernoulli_logit_glm_lpmf(
   static const char *function = "bernoulli_logit_glm_lpmf";
   typedef typename stan::partials_return_type<T_y, T_x, T_alpha, T_beta>::type
       T_partials_return;
+  typedef typename std::conditional<
+      is_vector<T_alpha>::value,
+      Eigen::Array<typename stan::partials_return_type<T_alpha>::type, -1, 1>,
```

Comment:
Is there a particular reason you're using `Eigen::Array` here instead of `Eigen::Matrix`?